crates/bevy_picking_input/src/touch.rs

```diff
@@ -0,0 +1,38 @@
+//! Provides sensible defaults for touch picking inputs.
+
+use bevy::{prelude::*, render::camera::RenderTarget, window::WindowId};
+use bevy_picking_core::pointer::{
+    InputMove, InputPress, Location, PointerButton, PointerId, PressStage,
+};
+
+/// Sends touch pointer events to be consumed by the core plugin
+pub fn touch_pick_events(
+    touches: Res<Touches>,
+    mut pointer_moves: EventWriter<InputMove>,
+    mut pointer_clicks: EventWriter<InputPress>,
+) {
+    for touch in touches.iter() {
+        if touch.distance() != Vec2::ZERO {
+            let id = PointerId::Touch(touch.id());
+            let location = Location {
+                target: RenderTarget::Window(WindowId::primary()),
```

Comment:
Guessing this does not support touch input across multiple windows just yet?

Reply:
Ah good catch.